cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -56,7 +56,6 @@
     if node.chef_environment == 'staging' && node.name == 'staging' # 'real' staging only
       cronjob at:'@reboot', do:home_dir('.dropbox-dist', 'dropboxd')
```

Comment:
can we drop this line, too?

Reply:
I don't fully know what this line does but I'm guessing the answer is yes 😁